desktop/TransactionManager.ts

```diff
@@ -300,48 +289,45 @@ class TransactionManager {
     data: Account__Output
   ) => {
     const currentState = {
-      counter: data.stateCurrent?.counter
-        ? data.stateCurrent.counter.toNumber()
-        : 0,
-      balance: data.stateCurrent?.balance?.value
-        ? data.stateCurrent.balance.value.toNumber()
-        : 0,
+      counter: data.stateCurrent?.counter?.toNumber?.() || 0,
+      balance: data.stateCurrent?.balance?.value?.toNumber?.() || 0,
```

Comment:
```suggestion
      balance: data.stateCurrent?.balance?.value?.toNumber() || 0,
```
maybe its redundent , on screenshot , its typescript issue , not fatal 
<img width="860" alt="Screenshot 2022-06-24 at 16 54 02" src="https://user-images.githubusercontent.com/54288612/175572230-dab6f4e6-f73c-4b39-aa3a-2c479d47224c.png">